### PR TITLE
3.9 Afternoon - Android Agent 6.10.0 release notes & doc updates

### PR DESCRIPTION
**Pull request by ywang-nr**
## Give us some context

* What problems does this PR solve?
   - Android agent v6.10.0 release notes
   - EOL updates
   - noticeNetworkFailure callout removal
   - New api shutdown() doc update
   - Clarify DT feature flag requests
   - Minor deprecation doc updates

### CODE REVIEW

**Comment by ywang-nr** [comment]
@cdillard-NewRelic for feedbacks

**Comment by ywang-nr** [comment]
@Doc Team, this PR is still in POC, feel free to add on any input/feedback to make it clear/better to users.

File: src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/shut-down.mdx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  - /docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/shut-down
+---
+
+## Syntax
+
+```
+NewRelic.shutdown()
+```
+
+Shut down the agent within the current application lifecycle during runtime
```
Comment (a-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/shut-down.mdx:20]:
```suggestion
Shut down the agent within the current application lifecycle during runtime.
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/shut-down.mdx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+## Requirements
+
+Agent version 6.10.0 or higher.
+
+## Description
+
+This API provides the ability to shutdown the agent within the current application lifecycle during runtime. Once the agent has shutdown, it cannot be restarted within the current application lifecycle.
```
Comment (a-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/shut-down.mdx:28]:
```suggestion
This API provides the ability to shut down the agent within the current application lifecycle during runtime. Once the agent has shut down, it cannot be restarted within the current application lifecycle.
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/shut-down.mdx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+For general info on using this API, see the [Android SDK API usage guide](/docs/mobile-monitoring/new-relic-mobile-android/install-configure/work-android-sdk-api).
```
Comment (a-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/shut-down.mdx:30]:
```suggestion
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/shut-down.mdx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+## Description
+
+This API provides the ability to shutdown the agent within the current application lifecycle during runtime. Once the agent has shutdown, it cannot be restarted within the current application lifecycle.
+
+
+
+## Examples
+
+### Shut down agent
```
Comment (a-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/shut-down.mdx:34]:
```suggestion
### Shut down the Android agent
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/shut-down.mdx [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+This API provides the ability to shutdown the agent within the current application lifecycle during runtime. Once the agent has shutdown, it cannot be restarted within the current application lifecycle.
+
+
+
+## Examples
+
+### Shut down the Android agent
+
+Example of shutting down the agent:
```
Comment (a-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/shut-down.mdx:36]:
```suggestion
Here's an example of shutting down the agent:
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/shut-down.mdx [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+```
+NewRelic.shutdown()
+```
```
Comment (a-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/shut-down.mdx:40]: @ywang-nr Can we build this example out more? Are there any scenarios where something would go in the brackets () of NewRelic.shutdown()?

File: src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/shut-down.mdx (hunk 2 above)
Comment (a-sassman) [reply src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/shut-down.mdx:28]: @ywang-nr What's a scenario where the user would want to use this API?

Comment (ywang-nr) [reply src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/shut-down.mdx]: This feature is meant to give the developer options to disable our agent, most likely related to their end users' privacy issue

File: src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/shut-down.mdx (hunk 6 above)
Comment (ywang-nr) [reply src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/shut-down.mdx:40]: This is all the developers need for shutting down the agent, no params are needed

**Review by ghost — DISMISSED** [review]
@ywang-nr Looks good to me! Let me know when you'd like to get this published.

**Comment by ywang-nr** [comment]
> @ywang-nr Looks good to me! Let me know when you'd like to get this published.

Awesome! Thank you so much for the feedback!
This PR will take some time to get merged as we don't have a target date to release this feature yet.

**Comment by ywang-nr** [comment]
@cthomas-newrelic for review

**Review by ghost — DISMISSED** [review]
@ywang-nr Approved! Let me know when it's ready to go live (or what time on 3/8).

**Comment by ywang-nr** [comment]
> @ywang-nr Approved! Let me know when it's ready to go live (or what time on 3/8).

Thank you, we will be pushing this back a few days maybe to next week, I will keep you posted. Thank you.

**Comment by ywang-nr** [comment]
@ally-sassman we're all ready to go for 3/9 afternoon merge, let me know if you have questions or concerns.

**Review by ghost — APPROVED** [review]

**Pull request merged**